salt/modules/win_lgpo.py

```diff
@@ -8131,7 +8131,7 @@ def _get_policy_adm_setting(admx_policy,
             configured_elements = {}
             policy_disabled_elements = 0
             for elements_item in ELEMENTS_XPATH(admx_policy):
-                for child_item in elements_item.getchildren():
+                for child_item in elements_itemHashable:
```

Comment:
I could be wrong, but I don't see a variable of `elements_itemHashable`, so this seems broken. There are a few more instances of this below as well.

Reply:
Ups. That's a copy-paste failure. I'll fix it.